PennMobile/Polls/PollsView.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  PollsView.swift
+//  PennMobile
+//
+//  Created by Jordan H on 9/10/23.
+//  Copyright © 2023 PennLabs. All rights reserved.
+//
+
+import SwiftUI
+
+struct PollsView: View {
+    @State var polls: [PollPost] = []
+    
+    var body: some View {
+        ScrollView {
+            VStack {
+                ForEach(polls, id: \.self.id) { poll in
+                    PollWrapper(pollQuestion: poll.poll, chosenId: poll.pollOptions[0].id)
+                }
+            }
+        }
+        .task {
+            let pollsResult = await PollsNetworkManager.instance.getPollHistory()
+            switch pollsResult {
+            case .failure(let error):
+                print(error)
+            case .success(let polls):
+                self.polls = polls
+            }
+        }
+    }
+}
+
+struct PollWrapper: UIViewRepresentable {
+    let pollQuestion: PollQuestion
+    let chosenId: Int?
+    var answeredPollQuestion: PollQuestion {
+        var question = pollQuestion
+        question.optionChosenId = chosenId
+        return question
+    }
+
+    func makeUIView(context: Context) -> HomePollsCell {
+        let cell = HomePollsCell()
+        cell.pollQuestion = answeredPollQuestion
+        cell.isUserInteractionEnabled = false
+        return cell
+    }
+    
+    @available(iOS 16.0, *)
+    func sizeThatFits(_ proposal: ProposedViewSize, uiView: HomePollsCell, context: Context) -> CGSize? {
+        return CGSize(width: proposal.width ?? 0, height: HomePollsCell.getPollHeight(for: answeredPollQuestion))
```

Comment:
Also I wanted to get rid of this "?? 0" and use the default size if possible, but I couldn't figure out how to. If nil is returned from this function, then the default size is used, but I couldn't figure out how to just set the height.